### DISCUSSION TITLE
Cider 2

### DISCUSSION THREAD

**Discussion by Flapperoo** [catppuccin/catppuccin#2396] [Port Requests]
### Will you make the port?

- [x] Yes, I'm willing to make the port

### How can the suggested port be themed?

Using the provided [themekit](https://github.com/ciderapp/cider-themekit) for Cider. Made by the same developers.

An example of a WIP Catppuccin Mocha in Cider 2 using the ThemeKit: 
![image](https://github.com/catppuccin/catppuccin/assets/22081694/e2fade31-b278-4c9b-be80-631672389778)

### Any additional comments?

The current [Catppuccin Cider repo](https://github.com/catppuccin/cider) is for an older and [deprecated](https://github.com/ciderapp/Cider?tab=readme-ov-file#-cider-classic-has-approached-its-end-of-life-status) version of Cider. This is aimed for the [current](https://cider.sh/) version utilizing the provided theming tools.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hey 👋 

Is there a reason this work can't be PR'd into the existing Cider repository to bring it up to date? I don't like the idea of having a catppuccin/cider repository and a catppuccin/cider-2 repository.

**Comment by sgoudham** [comment]
Closing as theming for Cider 2 is being tracked in https://github.com/catppuccin/cider/issues/1
